org.gnome.siglo.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id" : "org.gnome.siglo",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "siglo",
+    "finish-args" : [
+        "--allow=bluetooth",
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=system-bus",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "gatt",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
+            "build-commands": [
+                "pip3 install --prefix=/app --no-cache-dir gatt"
```

Comment:
...which mean that this won't work.

Nor the other python module.

Check https://github.com/flatpak/flatpak-builder-tools to assist building python modules.

Reply:
There's a helper tool for managing Python dependencies. It's documented [here](https://docs.flatpak.org/en/latest/python.html). Many Flathub repositories add the flatpak-builder-tools repo as a Git submodule. You can then execute the `flatpak-pip-generator` provided there, though you'll need to `pip install requirements-parser` first.

I have a project which uses the pip generator [here](https://github.com/jwillikers/i2cgui-flatpak). Hope it helps!

Reply:
once i figure out dbus stuff this is the next task,.

Reply:
done